### ISSUE TITLE
iosdfulibrary/iosdfulibrary-swift.h' file not found

### ISSUE THREAD

**Issue by dev-simpleplan** [Pilloxa/react-native-nordic-dfu#187]
![ios build issue](https://github.com/Pilloxa/react-native-nordic-dfu/assets/95477430/1e4befd4-f0a9-4204-b3be-144fe4269885)

I am facing some issues related to the IOSDFU library. How can we fix this issue?

**Comment by andresev** [comment]
@dev-simpleplan  Any luck on this? I'm getting the same error. I can't really find the ...-Swift.h file anywhere.

**Comment by dev-simpleplan** [comment]
> @dev-simpleplan Any luck on this? I'm getting the same error. I can't really find the ...-Swift.h file anywhere.

Yes, I have solved the issue. let me know if you are still facing the issue.

**Comment by Ritesh9811sahani** [comment]
![Screenshot 2024-08-27 at 12 25 02 PM](https://github.com/user-attachments/assets/f80d543c-b1fe-4677-a1f8-fb45807800ba)

This issue is still coming
